__test__/backend.test.js

```diff
@@ -1,7 +1,7 @@
-import { resolvers } from "../src/server/api/schema";
```

Comment:
General comment: Can you please use the prettier defaults (such as double quotes) so it's easier to see what has changed?

Reply:
Sure! Will do that. For some reason, husky & lint-staged are not running for me when I commit my stuff.